app/konnect/org-management/okta-idp.md

```diff
@@ -66,9 +66,7 @@ Create a new application in Okta to manage {{site.konnect_saas}} account integra
 
 ### Set up claims in Okta
 
-The connection between {{site.konnect_short_name}} and Okta uses OpenID Connect
-tokens. To have Okta send the correct information to your {{site.konnect_short_name}} org, set up
-claims to extract that information.
+This only needs to be followed if intending to use group claims for Konnect team mappings, this is an optional step. The connection between {{site.konnect_short_name}} and Okta uses OpenID Connect tokens. To have Okta send the correct information to your {{site.konnect_short_name}} org, set up claims to extract that information.
```

Comment:
```suggestion
If you are intending to use group claims for Konnect team mappings, follow this guide to set them up. Otherwise, skip to [Add a user to your application](#add-a-user-to-your-application).

The connection between {{site.konnect_short_name}} and Okta uses OpenID Connect tokens. To have Okta send the correct information to your {{site.konnect_short_name}} org, set up claims to extract that information.
```